docs/reference/aggregations/pipeline/change-point-aggregation.asciidoc

```diff
@@ -0,0 +1,98 @@
+[role="xpack"]
+[[search-aggregations-change-point-aggregation]]
+=== Change Point aggregation
+++++
+<titleabbrev>Change Point</titleabbrev>
+++++
+
+experimental::[]
+
+A sibling pipeline that detects a change point in a metric. Given a distribution of values provided by the sibling
+multi-bucket aggregation, this aggregation where find the bucket at which the distribution of points significantly
+changes.
+
+
+
+[[change-point-agg-syntax]]
+==== Parameters
+
+`buckets_path`::
+(Required, string)
+Path to the buckets that contain one set of values in which to detect a change point. There must be at least 21 bucketed
+values. Fewer than 1,000 is preferred.
+For syntax, see <<buckets-path-syntax>>.
+
+==== Syntax
+
+A `change_point` aggregation looks like this in isolation:
+
+[source,js]
+--------------------------------------------------
+{
+  "change_point": {
+    "buckets_path": "date_histogram>_count" <1>
+  }
+}
+--------------------------------------------------
+// NOTCONSOLE
+<1> The buckets containing the values to test against.
+
+[[change-point-agg-response]]
+==== Response body
+
+`bucket`::
+(Optional, object)
+Values of the bucket that indicates the discovered change point. Not returned if no change point was found.
+All the aggregations in the bucket are returned as well.
++
+.Properties of bucket
+[%collapsible%open]
+====
+`key`:::
+(value)
+The key of the bucket matched. Could be string or numeric.
+
+`doc_count`:::
+(number)
+The document count of the bucket.
+====
+
+`type`::
+(object)
+The found change point type and its related values. Possible types:
++
+--
+* `dip`: a significant dip occurs at this change point
+* `distribution_change`: the overall distribution of the values has changed significantly
+* `non_stationary`: there is no change point, but the values are not in a stationary distribution
+* `spike`: a significant spike occurs at this change point
+* `stationary`: no change point found
+* `step_change`: the change indicates a statistically significant step up or down in value distribution
+* `trend_change`: there is an overall trend change occurring at this change point
+--
+==== Response example
```

Comment:
This is not being formatted properly in the output. I think a blank line should fix:
```suggestion

==== Response example
```